src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -70,6 +70,9 @@
 ```
 Paste here the error message you have received. Make sure to remove all sensitive information,
 such as user name, password, credential, subscription ID, etc.
+
+The command failed with an unexpected error. Here is the traceback:
+{The rest of the error message}
```

Comment:
The error message starts with `The command failed with an unexpected error. Here is the traceback`. Hope it helps the user to copy the full error massage.